ProcessMaker/Listeners/UserSession.php

```diff
@@ -58,12 +46,12 @@ public function handle(object $event): void
             $user->sessions()
                 ->where([
                     ['is_active', true],
+                    ['ip_address', '!=', $ip],
                 ])
-                ->where(function (Builder $query) use ($agentDevice, $agentDeviceType, $agentPlatform, $ip) {
+                ->where(function (Builder $query) use ($agentDevice, $agentDeviceType, $agentPlatform) {
```

Comment:
Seems this could fail with two similar devices but with different IP addresses.

![image](https://github.com/ProcessMaker/processmaker/assets/8028650/50a4e72f-0a87-4906-948c-8a48555585ae)

Note that in this case the last login (id=13 for user=3 ip=192.168.1.163) is not killing the previous session (id=12 user=3 ip=192.168.1.119)

For the following configuration:

![image](https://github.com/ProcessMaker/processmaker/assets/8028650/b0be7885-0b76-4ef6-ba4b-edc88be481b5)

Reply:
Maybe you could consider to add a:
```
                        ->orWhere('ip_address', '!=', $ip)
```

Reply:
It was restored